tethys_gizmos/gizmo_options/map_view.py

```diff
@@ -219,6 +249,35 @@ class MapView(TethysGizmoOptions):
             legend_extent=[-173, 17, -65, 72]
         )
 
+        # Define base map options
+        esri_layer_names = [
+            'NatGeo_World_Map',
+            'Ocean_Basemap',
+            'USA_Topo_Maps',
+            'World_Imagery',
+            'World_Physical_Map',
+            'World_Shaded_Relief',
+            'World_Street_Map',
+            'World_Terrain_Base',
+            'World_Topo_Map',
+        ]
+        esri_layers = [{'ESRI': {'layer': l}} for l in esri_layer_names]
+        basemaps = [
+            'Stamen',
+            {'Stamen': {'layer': 'toner', 'label': 'Black and White'}},
+            {'Stamen': {'layer': 'watercolor'}},
+            'OpenStreetMap',
+            'CartoDB',
+            {'CartoDB': {'style': 'dark'}},
+            {'CartoDB': {'style': 'light', 'labels': False, 'label': 'CartoDB-light-no-labels'}},
```

Comment:
It’s somewhat confusing to have both the label and labels options. Perhaps we rename label to control_label or similar.